src/Hangfire.DynamicJobs/DynamicJob.cs

```diff
@@ -46,6 +48,11 @@ public sealed class DynamicJob
         [JsonProperty("f",  NullValueHandling = NullValueHandling.Ignore, ItemTypeNameHandling = TypeNameHandling.Auto)]
         public JobFilterAttribute[] Filters { get; }
 
+        //TODO: Check if is better save display name as resolved string (maybe when you use location resources can be cause crash?)
+        [CanBeNull]
+        [JsonProperty("dn", NullValueHandling = NullValueHandling.Ignore, ItemTypeNameHandling = TypeNameHandling.Auto)]
+        public JobDisplayNameAttribute DisplayName { get; }
```

Comment:
I think we should expose the `DisplayName` property as a `string` one and calculate its value when creating a dynamic job. Such descriptions are often based on argument values (by using placeholders like `{0}` passed to the `String.Format` method), and since argument types can be unavailable at runtime, we shouldn't depend on them.